server/channels/db/migrations/mysql/000138_add_remoteid_channelid_to_post_acknowledgements.up.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE PostAcknowledgements ADD COLUMN RemoteId varchar(26) DEFAULT '';
+ALTER TABLE PostAcknowledgements ADD COLUMN ChannelId varchar(26) NOT NULL DEFAULT '';
```

Comment:
We need to add the stored procedure check which checks if the column is there or not, and only adds if it is not there.